pkg/commands/workload_apply.go

```diff
@@ -49,8 +52,23 @@ var (
 	_ cli.DryRunable         = (*WorkloadApplyOptions)(nil)
 )
 
+const (
+	MergeUpdateStrategy   = "merge"
+	ReplaceUpdateStrategy = "replace"
```

Comment:
If no other files are using these, can it be private constants

Reply:
changed too